.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature Request 💡
+about: Suggest a new idea for the project.
+labels: "type: feature or enhancement"
```

Comment:
Does this need changed? 👀

Reply:
Maybe...I'd like to keep the labels for issues separate from what we label our PRs though, unless you think we should keep them the same?